General/Traits.md

```diff
@@ -38,7 +38,7 @@ Concrete example:
 Trait named: #FamixTWithEnumValues
 	uses: {}
 	slots: {}
-	category: 'Famix-Traits-EnumValue'
+	package: 'Famix-Traits-EnumValue'
```

Comment:
What is the first version of Pharo where this message (with `...package:`) is working?

Reply:
It was introduced in Pharo 8. I propose to add a note with the API from before Pharo 8.